### PR TITLE
Added localization support on all the pages

### PR DESCRIPTION
**Pull request by AndreaLandiArk**
Refactor directories tree structure

### CODE REVIEW

File: public/locales/en/zodCustom.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
     "custom_error": "This is a custom error.",
+    "password_complexity": "password needs to be at least 6 characters long",
     "custom_error_with_path": "'{{- path}}' is not valid. This is a custom error.",
     "paths": {
-        "customErrorInline": "The Translated Field Name"
+        "customErrorInline": "The Translated Field Name",
+        "phone": "Invalid Phone Number",
```
Comment (AndreaCuneo) [inline public/locales/en/zodCustom.json:7]: I don't like this. 'paths' is only to translate the name of the field, not to specify the problem.

File: public/locales/it/zodCustom.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "custom_error": "Questo è un messaggio custom.",
+    "password_complexity": "La password deve contenere almeno 6 caratteri.",
     "custom_error_with_path": "'{{- path}}' non va bene. Questo è un messaggio custom.",
     "paths": {
```
Comment (AndreaCuneo) [inline public/locales/it/zodCustom.json:5]: translation for fields missing in respect to en

File: src/components/header/view.tsx [hunk 3]
```diff
@@ -20,11 +20,12 @@ import { useAppDispatch } from "../../app/hooks";
 import {
   Login,
   Logout,
-} from "../../features/authentication/authenticationSlice";
-import { useAuthContext } from "../../lib/authentication/useAuthContext";
+} from "../../lib/authentication/redux/authenticationSlice/authenticationSlice";
```
Comment (AndreaCuneo) [inline src/components/header/view.tsx:23]: toooo long.

/lib/authentication/authenticationSlice.ts suffices

**Review by AndreaCuneo — CHANGES_REQUESTED** [review]

File: public/locales/en/zodCustom.json (hunk 1 above)
Comment (AndreaLandiArk) [reply public/locales/en/zodCustom.json:7]: this was a leftover

**Review by AndreaCuneo — APPROVED** [review]

File: src/components/header/view.tsx [hunk 4]
```diff
@@ -123,6 +121,9 @@ const Header = () => {
         <Spacer />
         <Spacer />
         <Spacer />
+        <Center mr={"20px"}>
+          <LocaleSwitcher />
```
Comment (AndreaCuneo) [inline src/components/header/view.tsx:125]: skip this if supportedLangs is empty or contains a single lang. No switch if nothing to switch.

I'd suggest to code the 'if' within the LocaleSwitcher

**Review by AndreaCuneo — CHANGES_REQUESTED** [review]

**Review by AndreaCuneo — APPROVED** [review]

**Pull request merged**